src/unix/proc.c

```diff
@@ -64,7 +64,7 @@ procRun(unsigned nargs, char *const args[], int *rc)
 	extern char   **environ;
 	int		ret;
 	int		child;
-	char		so        [PATH_MAX];
+	char		so        [PATH_MAX+3];
```

Comment:
Why this?

Reply:
I got a linter warning because of `snprintf(so, sizeof(so), "%s.so", fullpath);`, so has better be 3 characters longer than fullpath

Reply:
I see, clang static-analyzer, right? Fine then.